### PR TITLE
Update hpc-coding-convention

### PR DESCRIPTION
**Pull request by alkino**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -59,10 +59,10 @@ set(CORENRN_ClangFormat_FILES_RE
     "^.*\\\\.cu$$" "^.*\\\\.[chi]pp$$"
     CACHE STRING "List of regular expressions matching C/C++ filenames" FORCE)
 set(CORENRN_ClangFormat_EXCLUDES_RE
-    "${CORENRN_PROJECT_SOURCE_DIR}/external/.*$$"
+    "external/.*$$"
```
Comment (olupton) [inline CMakeLists.txt:62]:
```suggestion
    ""
```

File: CMakeLists.txt [hunk 2]
```diff
@@ -59,10 +59,10 @@ set(CORENRN_ClangFormat_FILES_RE
...
     CACHE STRING "list of regular expressions to exclude C/C++ files from formatting" FORCE)
 set(CORENRN_CMakeFormat_EXCLUDES_RE
-    "${CORENRN_PROJECT_SOURCE_DIR}/external/.*$$" ".*/CMake/packages/.*$$"
+    "external/.*$$" "CMake/packages/.*$$"
```
Comment (olupton) [inline CMakeLists.txt:65]:
```suggestion
    "CMake/packages/.*$$"
```

**Review by olupton — COMMENTED** [review]
I don't think we need to exclude submodule directories any more, do we?

File: CMakeLists.txt (hunk 1 above)
Comment (tristan0x) [reply CMakeLists.txt:62]: that's right, paths are relative to ${CORENRN_PROJECT_SOURCE_DIR} now

**Review by olupton — APPROVED** [review]

**Pull request merged**